hummingbot/client/performance.py

```diff
@@ -112,15 +112,5 @@ def divide(value, divisor):
 
 
 def smart_round(value: Decimal) -> Decimal:
-    step = Decimal("1")
-    if Decimal("10000") > abs(value) > Decimal("100"):
-        step = Decimal("0.1")
-    elif Decimal("100") > abs(value) > Decimal("1"):
-        step = Decimal("0.01")
-    elif Decimal("1") > abs(value) > Decimal("0.01"):
-        step = Decimal("0.0001")
-    elif Decimal("0.01") > abs(value) > Decimal("0.0001"):
-        step = Decimal("0.00001")
-    elif Decimal("0.0001") > abs(value) > s_decimal_0:
-        step = Decimal("0.000001")
-    return (value // step) * step
+    # Simply quantize values to 8 decimal places
+    return Decimal(str(value)).quantize(Decimal("0.00000001"))
```

Comment:
if we do this, it will no longer be a smart round, also what happens when this fn is used on large value e.g. BTC-USDT of 14,000?

Reply:
Yeah, I agree it may no longer be a smart function. If that's the case I can do a rename to `report_round`, just felt that wasn't necessary. A value of 14,000 would be 14000.00000000.